packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -245,7 +245,20 @@ const makeRenderer = (
 					initialNow: new Date().getTime(),
 					isProdApi: server.settings.app.api.isProd,
 					isQL: parseMemberCookie(state).ql === 'true',
-					memberId: parseMemberCookie(state).id,
+					/**
+					 * Original tickets that caused need:
+					 * 
+					 * @see https://meetup.atlassian.net/browse/MG-2473
+					 * @see https://meetup.atlassian.net/browse/MG-2441
+					 * 
+					 * For segmentation and rollout purposes, it is extremely advantageous
+					 * to have the member object available pre query to do split tests
```

Comment:
the member cookie is not the same as the member object returned by `/members/self`, so be careful if you are relying on it having the same properties in any downstream code